src/routes/assistant/[assistantId]/+page.server.ts

```diff
@@ -0,0 +1,20 @@
+import { base } from "$app/paths";
+import { collections } from "$lib/server/database.js";
+import { redirect } from "@sveltejs/kit";
+import { ObjectId } from "mongodb";
+
+export const load = async ({ params }) => {
+	try {
+		const assistant = await collections.assistants.findOne({
+			_id: new ObjectId(params.assistantId),
+		});
+
+		if (!assistant) {
+			throw redirect(302, `${base}`);
+		}
+
+		return { assistant: JSON.parse(JSON.stringify(assistant)) };
```

Comment:
```suggestion
		return { assistant };
```

is it not possible to do so?

Reply:
The main reason why this is a thing is that mongodb returns SOME `ObjectId` for the IDs which are not happy being serialized by sveltekit. If I remove this line I get the following error:

```
Error: Data returned from `load` while rendering /assistant/[assistantId] is not serializable: Cannot stringify arbitrary non-POJOs (data.assistant._id)
``` 

If you know a better way to handle this let me know, I would happily get rid of this line :grin:

Reply:
oh I see